tests/model_serving/model_server/authentication/conftest.py

```diff
@@ -135,6 +210,29 @@ def patched_remove_authentication_isvc(
         yield http_s3_caikit_serverless_inference_service
 
 
+@pytest.fixture()
+def patched_remove_raw_authentication_isvc(
+    admin_client: DynamicClient,
+    http_s3_caikit_raw_inference_service: InferenceService,
+) -> InferenceService:
+    with ResourceEditor(
+        patches={
+            http_s3_caikit_raw_inference_service: {
+                "metadata": {
+                    "labels": {"security.opendatahub.io/enable-auth": "false"},
+                }
+            }
+        }
+    ):
+        predictor_pod = get_pods_by_isvc_label(
+            client=admin_client,
+            isvc=http_s3_caikit_raw_inference_service,
+        )[0]
+        predictor_pod.wait_deleted()
```

Comment:
when auth is disabled, does the pod restart?
it was fixed in kserve and i think this should be the case here as well

Reply:
its a case of authorino (Server-less  not raw )